packages/@atjson/document/src/annotation.ts

```diff
@@ -275,26 +276,17 @@ export abstract class Annotation<Attributes = {}> {
       // with the preserveAdjacentBoundaries switch.
 
       // Default edge behaviour.
-    } else if (change.behaviour === AdjacentBoundaryBehaviour.default) {
```

Comment:
The real meat of the change starts here, inverted the logic a little bit to make it clearer